vmas/scenarios/navigation_obstacles.py

```diff
@@ -72,6 +73,10 @@ def make_world(self, batch_dim: int, device: torch.device, **kwargs):
         )
         entity_filter_agents: Callable[[Entity], bool] = lambda e: isinstance(e, Agent) or isinstance(e, Landmark) and e.collide
 
+        # if running in alcove mode, ensure only 2 agents
+        if (self.map_name == "alcove"):
+            self.n_agents = 2
```

Comment:
```suggestion
            assert self.n_agents == 2, "Alcove env requires 2 agents"
```
I tried setting n_agents=4 but running alcove, and the disagreement between number of agents trained + number of agents expected by the environment created a confusing error message. Recommend simply asserting with a clear error message like this.

Reply:
oh this makes a lot more sense, ill do that lol